openedx/core/djangoapps/courseware_api/tests/test_views.py

```diff
@@ -130,12 +130,14 @@ def test_course_metadata(self, logged_in, enrollment_mode, enable_anonymous, is_
                                               'The audit track does not include a certificate.')
                     assert response.data['certificate_data']['msg'] == expected_audit_message
                     assert response.data['verify_identity_url'] is None
+                    assert response.data['verification_status'] is 'none'
                     assert response.data['linkedin_add_to_profile_url'] is None
                 else:
                     assert response.data['certificate_data']['cert_status'] == 'earned_but_not_available'
                     expected_verify_identity_url = reverse('verify_student_verify_now', args=[self.course.id])
                     # The response contains an absolute URL so this is only checking the path of the final
                     assert expected_verify_identity_url in response.data['verify_identity_url']
+                    assert response.data['verification_status'] is 'none'
```

Comment:
I know this is quick fix, but in an ideal world, we'd have a test that checked for a non-none status.

Reply:
I'm with you, but I'm treating this as having those cases tested by the method I'm calling out to so it's not _as_ necessary to test again here. That and setting up all of the cases to have this return a not `'none'` status is a lot.